src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 23px 0;
+  margin: 0;
+}
+
+.image {
+  display: flex;
+  padding: 10px 0 10px 50px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  padding: 23px 0;
+  margin-left: 20px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__item .nav__link:first-of-type {
+  margin: 0;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```

Comment:
it's better to move colors into variables

Reply:
I don't know yet, how it works.